cluster/gce/gci/configure-helper.sh

```diff
@@ -95,7 +95,7 @@ function main() {
     gke-master-start
   else
     create-node-pki
-    if [[ "${KUBERNETES_RESOURCE_PARTITION:-false}" == "true" ]] || [[ "${KUBERNETES_TENANT_PARTITION:-false}" == "true" ]]; then
+    if [[ "${ENABLE_APISERVER_INSECURE_PORT:-false}" == "true" ]]; then
```

Comment:
ENABLE_INSECURE_SCALEOUT_CLUSTER_MODE?

Reply:
fixed. note this is outdated in the conversation. in the code review section, this is fixed already